app/models/foram.rb

```diff
@@ -23,7 +23,7 @@ def self.for_user(user)
     session = user.try { |user| user.mongo_sessions.active.last }
     if session
       session.connect
-      with(session: session.config_name).all
+      with(session: session.config_name, collection: session.foram_collection).all
```

Comment:
Trochę szkoda, że dla każdej kolekcji trzeba tworzyć osobną sesje. Fajnie by było dodać gdzieś jakiś dropdown z którego można by wybrać kolekcje z danej bazy danych (dodać jakiś improvement na jirze). Póki co możemy poczekać na to co Topa powie.